File: client/src/models/shopItem.ts
import { IngredientItem } from "./ingredient";

export type ShopItem = {
  id: string;
  description?: string;
  recentlyUsed: number;
  categoryId: string;
  ingredientId: string;
  ingredientTypeId?: string;
  name: string;
  order?: number;
};

File: server/Entities/ShopItem.cs
namespace MiddagApi.Models
{
  public class ShopItem
  {
    public string id { get; set; }
    public string name { get; set; }
    public string categoryId { get; set; }
    public string ingredientId { get; set; }
    public string? ingredientTypeId { get; set; }
    public string? description { get; set; }
    public long? recentlyUsed { get; set; }
    public long? order { get; set; }
  }
}